warehouse/templates/rss/user_updates.xml

```diff
@@ -0,0 +1,13 @@
+{% extends "base.xml" %}
+{% block title %}{{ user.name }} recent updates{% endblock %}
```

Comment:
This should be `username` it's currently not rendering anything resulting `<title> recent updates</title>`

```suggestion
{% block title %}{{ user.username }} recent updates{% endblock %}
```

Also happening in the line below.

Reply:
Fixed.